### DISCUSSION TITLE
Can the new Dashboard or serverless shiny app functionality be embedded in a quarto book?

### DISCUSSION THREAD

**Discussion by usrbinr** [quarto-dev/quarto-cli#8218] [Q&A] [answered]
### Description

hi!

I'm really excited about the new Dashboard and shinyless server functionality and the possibilities with Quarto. 

Does anyone know if either the [dashboard](https://quarto.org/docs/dashboards/)  or serverless shiny capabilities can be embedded in a Quarto Book format (eg. say as a dedicated chapter or part of the page)?

**Comment by mcanouil** [comment, marked as answer]
Shiny is independent of Dashboard and can be used in any HTML component, see the documentation: <https://quarto.org/docs/interactive/shiny/index.html>.

Dashboard is a format on its own, so it could be linked or embedded as any HTML components but not "combined".
